### PR TITLE
Adding more instance types

### PR DESCRIPTION
**Pull request by ramsrivatsa**
Adding more instance type definitions

### CODE REVIEW

File: service_capacity_modeling/capacity_planner.py [hunk 1]
```diff
@@ -189,13 +189,25 @@ def _set_instance_objects(
...
-            zonal_cluster_capacity.cluster_instance = hardware.instances[
-                zonal_cluster_capacity.cluster_instance_name
-            ]
+            if zonal_cluster_capacity.cluster_instance_name in hardware.instances:
+                zonal_cluster_capacity.cluster_instance = hardware.instances[
+                    zonal_cluster_capacity.cluster_instance_name
+                ]
+            else:
+                raise ValueError(
+                    f"Model not trained to right size cass clusters that are of "
```
Comment (arunagrawal-84) [inline service_capacity_modeling/capacity_planner.py:198]: Isn't this in generic interface?

File: service_capacity_modeling/hardware/profiles/shapes/aws.json [hunk 2]
```diff
@@ -332,7 +340,41 @@
...
       }
     },
+    "i4i.16xlarge": {
+      "name": "i4i.16xlarge",
+      "cpu": 64,
+      "cpu_ghz": 3.5,
+      "ram_gib": 495.82,
+      "net_mbps": 35000,
+      "drive": {
+        "name": "ephem", "size_gib": 3492,
```
Comment (arunagrawal-84) [inline service_capacity_modeling/hardware/profiles/shapes/aws.json:350]: i4i.16xl is 13968 GiB and can you also update i4i.8xl ephem is 6984GiB

File: service_capacity_modeling/hardware/profiles/shapes/aws.json [hunk 3]
```diff
@@ -379,6 +421,18 @@
         "block_size_kib": 4, "single_tenant": true
       }
     },
+    "i3en.6xlarge": {
+      "name": "i3en.6xlarge",
+      "cpu": 24,
+      "cpu_ghz": 3.1,
+      "ram_gib": 186.62,
+      "net_mbps": 12000,
```
Comment (arunagrawal-84) [inline service_capacity_modeling/hardware/profiles/shapes/aws.json:429]: Net_mbps is 24000

File: service_capacity_modeling/capacity_planner.py (hunk 1 above)
Comment (ramsrivatsa) [reply service_capacity_modeling/capacity_planner.py:198]: Yes most of them are. The one's that arent we'd rather handle this much more gracefully than an index out-of-bounds error.

Comment (arunagrawal-84) [reply service_capacity_modeling/capacity_planner.py:198]: Can we remove the word "cass" then?

File: service_capacity_modeling/capacity_planner.py [hunk 4]
```diff
@@ -189,13 +189,25 @@ def _set_instance_objects(
...
-            regional_cluster_capacity.cluster_instance = hardware.instances[
-                regional_cluster_capacity.cluster_instance_name
-            ]
+            if regional_cluster_capacity.cluster_instance_name in hardware.instances:
+                regional_cluster_capacity.cluster_instance = hardware.instances[
+                    regional_cluster_capacity.cluster_instance_name
+                ]
+            else:
+                raise ValueError(
+                    f"Model not trained to right size cass clusters that are of "
```
Comment (arunagrawal-84) [inline service_capacity_modeling/capacity_planner.py:208]: Remove "cass"

**Pull request merged**